python/lsst/ts/Dome/dome_csc.py

```diff
@@ -506,10 +519,33 @@ def remove_keys_from_dict(self, dict_with_too_many_keys, keys_to_remove):
         dict_with_keys_removed = {
             x: dict_with_too_many_keys[x]
             for x in dict_with_too_many_keys
-            if x not in keys_to_remove
+            if x not in _KEYS_TO_REMOVE
         }
         return dict_with_keys_removed
 
+    # noinspection PyMethodMayBeStatic
+    def convert_telemetry_radians_to_degrees(self, telemetry_in_radians):
+        telemetry_in_degrees = {}
+        for key in telemetry_in_radians.keys():
+            if key in _KEYS_IN_RADIANS:
+                telemetry_in_degrees[key] = (
+                    telemetry_in_radians[key] * _RADIANS_TO_DEGREES
+                )
+            else:
+                telemetry_in_degrees[key] = telemetry_in_radians[key]
+        return telemetry_in_degrees
+
+    # noinspection PyMethodMayBeStatic
+    def convert_telemetry_list_radians_to_degrees(self, telemetry_in_radians):
+        telemetry_in_degrees = {}
+        for key in telemetry_in_radians.keys():
+            if key in _KEYS_IN_RADIANS:
+                arr = np.array(telemetry_in_radians[key])
+                telemetry_in_degrees[key] = (arr * _RADIANS_TO_DEGREES).tolist()
+            else:
+                telemetry_in_degrees[key] = telemetry_in_radians[key]
+        return telemetry_in_degrees
```

Comment:
@pkubanek 's suggestion to use `numpy.degrees` would work well here, since I think you can use the same code for both lists and scalars

That only works if you are willing to have the values be numpy arrays instead of lists. But if that works then I would argue it's actually better to have the values be numpy arrays.

Reply:
Unfortunately I cannot use numpy arrays everywhere mostly because yaml doesn't handle them well and lists of values are needed for the communication with the lower level components. I will modify the code to use numpy for the list conversion though since it is much simpler and clearer. Also, I got rid of the constants and am using the math.radians and math.degrees functions now everywhere since I think that is clearer and more concise as well.